subprojects/packagefiles/libsndfile/meson.build

```diff
@@ -104,11 +104,12 @@ sndfile = configure_file(
 
 incdirs = include_directories('.', 'src')
 
+pymod = import('python')
 def = custom_target(
   'def',
   output: 'sndfile.def',
```

Comment:
This could in theory just not be built on Linux, idk.

The bigger issue, is that you actually need to run it on Linux too, to create a GNU ld version script. It sets up symbol versioning e.g. `sf_close@@libsndfile.so.1.0`